### DISCUSSION TITLE
Consent Management

### DISCUSSION THREAD

**Discussion by gjmveloso** [hugo-sid/hugo-blog-awesome#267] [Feature requests]
With native Google Analytics integration it may be good to consider adding a [consent management](https://support.google.com/analytics/answer/14009343?sjid=6656248734750142849-SA#web-consent) overlay to be in compliance while serving traffic to European Economic Area (EEA) users.

**Comment by hugo-sid** [comment]
Thank you for the note. I have a different plan though.

I am considering removing the GA integration altogether. 

Maybe, this can just go in README (under heading like How to setup GA).
